upup/pkg/fi/cloudup/awsup/aws_cloud.go

```diff
@@ -40,6 +42,7 @@ import (
 	"k8s.io/kops/upup/pkg/fi"
 	"k8s.io/kubernetes/federation/pkg/dnsprovider"
 	dnsproviderroute53 "k8s.io/kubernetes/federation/pkg/dnsprovider/providers/aws/route53"
+	k8s_aws "k8s.io/kubernetes/pkg/cloudprovider/providers/aws"
```

Comment:
Let's just copy it in, not add another k/k dependency

Reply:
Actually on seconds thoughts it is such a beast it's probably better not to copy it until we have to...

Reply:
SGTM ... yes beast